conda/environments/all_cuda-118_arch-x86_64.yaml

```diff
@@ -65,7 +65,7 @@ dependencies:
 - nvcomp==3.0.5
 - nvtx>=0.2.1
 - packaging
-- pandas>=2.0,<2.1.5dev0
```

Comment:
Is this the right pinning, or should we go with `<2.3.0dev0`? Do we expect patch releases to break us?

Reply:
I feel various fixes are going into `2.2.1` that may have an impact on us, but would defer to @mroeschke on that.

Reply:
Yeah I fixed various issues @galipremsagar found in pandas 2.2.1 (coming out next week) which there are xfails for so the update will break the test suite. We should probably relax the pin in a follow up PR when that's out